packages/perseus/src/init.ts

```diff
@@ -2,57 +2,19 @@ import basicWidgets from "./basic-widgets";
 import extraWidgets from "./extra-widgets";
 import * as Widgets from "./widgets";
 
-declare const MathJax: any;
-
-export type PerseusOptions = {
-    // TODO(LEMS-1608): remove skipMathJax once we have completely removed the
-    // legacy MathJax 2 renderer from webapp.
-    skipMathJax: boolean;
-};
-
 /**
  * This should be called by all clients, specifying whether extra widgets are
  * needed via `loadExtraWidgets`. It is idempotent, so it's not a problem to
  * call it multiple times.
- *
- * skipMathJax:
- *   If false/undefined, MathJax will be configured, and the
- *   promise will wait for MathJax to load (if it hasn't already).
  */
-const init = function (options: PerseusOptions): Promise<undefined> {
+const init = function (): Promise<undefined> {
```

Comment:
Given that there's nothing async in this function anymore, can you make it just `const init = function() {`?